substrate/primitives/core/src/crypto.rs

```diff
@@ -841,7 +828,6 @@ impl sp_std::str::FromStr for SecretUri {
 /// Trait suitable for typical cryptographic PKI key pair type.
 ///
 /// For now it just specifies how to create a key from a phrase and derivation path.
-#[cfg(feature = "full_crypto")]
```

Comment:
`bls` and `paired_crypto` primitives need the same tweaks applied to the other primitives. 
(e.g. always implement `Pair` and only gate `sign` behind `full_crypto` feat

Reply:
@davxy: done in [6cb449a](https://github.com/paritytech/polkadot-sdk/pull/2044/commits/6cb449a49c4d6f78912d9ffd673e36b51213f48c), would you please take a look?